### PR TITLE
tests: Only test help.1

### PR DESCRIPTION
**Pull request by pkubatrh**
same PR as sclorg/mariadb-container#35

### CODE REVIEW

File: 4.1/test/run [hunk 1]
```diff
@@ -52,18 +52,16 @@ run_doc_test() {
   local tmpdir=$(mktemp -d)
   local f
   echo "  Testing documentation in the container image"
-  # Extract the help files from the container
-  for f in /usr/share/container-scripts/ror/README.md help.1 ; do
```
Comment (pvalena) [inline 4.1/test/run]: Why not keep the `for ` syntax in case we need testing other files?

Comment (pkubatrh) [reply 4.1/test/run]: That is a good point actually, thanks.

**Comment by pkubatrh** [comment]
Modified the PR to leave the `for` loop in place in case we need to test multiple files in future

**Comment by pvalena** [comment]
LGTM, Thanks!

**Pull request merged**